config/projects/chefdk.rb

```diff
@@ -65,6 +65,7 @@
 # Going to leave 2.1.5 for now since there is a workaround
 override :'ruby-windows', version: "2.1.6"
 override :'ruby-windows-devkit', version: "4.7.2-20130224-1151"
+override :'openssl-windows', version: "1.0.1m"
```

Comment:
Do we need that same pinning in Chef? If so should we just update `default_version` in the openssl-windows software def to the latest?